devtools/resetdb/main.go

```diff
@@ -277,6 +277,10 @@ func fillDB(ctx context.Context, url string) error {
 	dt.Wait()
 	_, err = pool.Exec(ctx, "alter table alerts enable trigger all")
 	must(err)
+
+	// fix sequences
+	_, err = pool.Exec(ctx, "SELECT pg_catalog.setval('public.alerts_id_seq', (select count(*) from public.alerts), true);")
```

Comment:
Shouldn't this grab the max + 1? That way future changes don't assume sequential ordering